README.md

```diff
@@ -1,13 +1,34 @@
-# Happy Thoughts
+# Happy Thoughts [View Live &#10555;](https://dontworry-behappy.netlify.app/)
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+In this project I have created a "twitter-like" app in React which is fetching and posting data to an API, using useState and useEffect.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started off by building the two components with API and then added code according to instructions and requirements:
+  * Made a form to post new thoughts.
+  * Made the app show all the written messages and how many likes each has recieved.
+  * Listed the latest message on top, oldest in the bottom (sorted).
+  * Created a heart-button for users to send likes on each thought.
+  * Made the colour of the button vary depending on if it has got any likes or not.
+  * Followed a given design-guideline.
+
+  * I also made a counter for letters written in the form.
+
+ Next steps would be:
+  * Create warning if the message written in the form is too long.
+  * Error-message if sent message is too long/ short/ empty (friendly message).
+  * Create Handle loading states
+  * Cunter for how many different posts you have liked 
+  * Animation for new thought added
+
+## TECH used:
+ * React
+ * useEffect hook
+ * useState hook
+ * Fetch
+ * Postman
+ * CSS
 
 ## View it live
+https://dontworry-behappy.netlify.app/
 
```

Comment:
This is the best and most clear ReadMe I have read so far. Very useful for someone reading the code. 👍